src/patternfly/components/Page/examples/Page.md

```diff
@@ -205,6 +208,7 @@ This component provides the basic chrome for a page, including sidebar, header,
 | `.pf-c-page__main-nav` | `<section>` |   Creates a container to nest the navigation component in the main page area. |
 | `.pf-c-page__main-breadcrumb` | `<section>` |   Creates a container to nest the breadcrumb component in the main page area. |
 | `.pf-c-page__main-section` | `<section>` |  Creates a section container in the main page area. **Note: The last/only `.pf-c-page__main-section` element will grow to fill the availble vertical space. You can change this behavior using `.pf-m-fill` and `.pf-m-no-fill`, which are documented below.**  |
+| `.pf-c-page__main-tabs` | `<section>` | Creates a container to nest the tabs component in the main page area. |
```

Comment:
Is this specific section needed? Per @gdoyle1's [comment ](https://github.com/patternfly/patternfly/pull/3914#issuecomment-797554356) it doesn't look like we need that page section, and can just use `.pf-c-page__main-section` instead.

> Tabs in page section -- we don't expect anyone to use tabs without a page section, so I think it's fine! We'll have that detail in the design guidelines as well, that the tabs should be placed underneath a title on the page.


The problem with it is that we can't guarantee users will use the right page section to wrap the tabs. `.pf-c-page__main-[nav/breadcrumbs]` work because the tertiary nav and breadcrumbs are props on the `<Page>` component, so react can put the nav/breadcrumbs in the appropriate page section for the user. Since we don't know where tabs will go on the page, I don't think we can support adding tabs to a page via a prop, which means users will need to know to put it in `.pf-c-page__main-tabs.` I just wonder if it will lead to inconsistencies.